source/extensions/extensions_build_config.bzl

```diff
@@ -197,6 +197,7 @@ EXTENSIONS = {
     #
 
     "envoy.transport_sockets.alts":                     "//source/extensions/transport_sockets/alts:config",
+    "envoy.transport_sockets.s2a":                      "",
```

Comment:
Let's add a TODO to add the config target here so that we don't forget about this. Thanks.